examples/gallery/symbols/patterns.py

```diff
@@ -0,0 +1,91 @@
+r"""
+Bit and Hachure Patterns
+------------------------
+
+PyGMT allows using bit or hachure patterns via the ``fill`` parameter
+or similar parameters:
+
+- :meth:`pygmt.Figure.coast`: Land and water masses via ``land`` and
+  ``water``, respectively
+- :meth:`pygmt.Figure.histogram`: Histogram bars via ``fill``
+- :meth:`pygmt.Figure.meca`: Focal mechanisms via ``compressionfill``
+  and ``extensionfill``
+- :meth:`pygmt.Figure.plot`: Symbols and polygons via ``fill``
+- :meth:`pygmt.Figure.rose`: Histogram sectors via ``fill``
+- :meth:`pygmt.Figure.solar`: Day-light terminators via ``fill``
+- :meth:`pygmt.Figure.ternary`: Symbols via ``fill``
+- :meth:`pygmt.Figure.velo`: Uncertainty wedges and velocity error
+  ellipses via ``uncertaintyfill``
+- :meth:`pygmt.Figure.wiggle`: Anomalies via ``fillpositive``
+  and ``fillnegative``
+
+The required argument has the following form:
+
+**P**\ |**p**\ *pattern*\ [**+b**\ *color*\ ][**+f**\ *color*\ ][**+r**\ *dpi*]
+
+*pattern* can either be a number in the range 1â€“90 or the name of a
+1-, 8-, or 24-bit image raster file. The former will result in one of the 90
+predefined 64 x 64 bit-patterns provided with GMT; an overview can by found at
```

Comment:
```suggestion
predefined 64 x 64 bit-patterns provided by GMT; an overview can by found at
```

Reply:
From my side, this change is fine. I orientated myself on the formulation in the upstream GMT documentation at https://docs.generic-mapping-tools.org/dev/cookbook/features.html#specifying-area-fill-attributes.